styleguide/_themes/derek/scss/components/_basement.scss

```diff
@@ -0,0 +1,5 @@
+.basement{
+  &, *{
```

Comment:
Will the basement be continued in another PR? If not this should be refactored now.